live-examples/js-examples/classes/classes-static.js

```diff
@@ -1,8 +1,13 @@
 class ClassWithStaticMethod {
+
+  static staticProperty = "someValue";
```

Comment:
```suggestion
  static staticProperty = 'someValue';
```

…otherwise the double-quotes trigger an error from the linter — see https://travis-ci.org/github/mdn/interactive-examples/builds/734582177#L229